kernel/kernel-benchmarks/src/test/java/io/delta/kernel/benchmarks/models/CCv2Info.java

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.kernel.benchmarks.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.delta.kernel.engine.Engine;
+import io.delta.kernel.utils.FileStatus;
+import io.delta.storage.commit.Commit;
+import io.delta.unity.InMemoryUCClient;
+import io.delta.unity.UCCatalogManagedCommitter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import scala.collection.mutable.ArrayBuffer;
+
+/**
+ * Represents CCv2 (Coordinated Commits v2) configuration for a Delta table used in benchmarks.
```

Comment:
Let's also not say `Coordinated Commits`. That's the old and rejected table feature name.